tests/forte/data/data_store_test.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for data store related operations.
+"""
+
+import logging
+import unittest
+from sortedcontainers import SortedList
+
+from forte.data.data_store import DataStore
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class DataStoreTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data_store = DataStore()
+        # attribute fields for Document and Sentence entries
+        self.data_store._type_attributes = {
+            "Document": ["document_class", "sentiment", "classifications"],
+            "Sentence": [
+                "speaker",
+                "part_id",
+                "sentiment",
+                "classification",
+                "classifications",
+            ],
+        }
+        # The order is [Document, Sentence]. Initialize 2 entries in each list.
+        # Document entries have tid 1234, 3456.
+        # Sentence entries have tid 9999, 1234567.
+        # The type id for Document is 0, Sentence is 1.
+        self.data_store.elements = [
+            SortedList(
+                [
+                    ["Document", 1234, 0, 5, None, "Postive", None],
+                    [
+                        "Document",
```

Comment:
What's the purpose of storing "Document" value here?

Reply:
> What's the purpose of storing this string "Document" here?

When user use only tid to refer to an entry, we want to know the type, so we also store it.

Reply:
I see, note (as in a previous comment), this need to be a full qualified class name